### PR TITLE
New Filter/Search Next/Back button #1216

### PR DESCRIPTION
**Pull request by liaojikunwork**

### CODE REVIEW

**Comment by pimato** [comment]
the divider should be removed.
Also if a button is active the icon should not be blue as it is right now.

**Review by pimato — DISMISSED** [review]
check comment

**Review by pimato — APPROVED** [review]

File: app/src/theme/index.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ const theme = {
   colors: {
     activeListItemBackground: '#fafafa',
     alert: '#E57373',
+    borderColorDar: '#9FA8DA',
```
Comment (KadenZipfel) [inline app/src/theme/index.js:79]: Looks like a typo here :p

Comment (liaojikunwork) [reply app/src/theme/index.js]: Just fixed and pushed again.

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**